docs/integration/ibm-cloud-kubernetes-logs.md

```diff
@@ -0,0 +1,253 @@
+title: IBM Cloud Kubernetes Logs Integration
+description: Sematext IBM Cloud Kubernetes Logs integration is configured by running Logagent as a Daemonset in your cluster.
+
+With this integration you can:
+
+- Forward all Containerd container logs
+- Use log globs to choose which containers to log
```

Comment:
"which containers to log" sounds a little funky

Reply:
Changed to: "Use log globs to choose which container log files to tail"